continuous_integration/hdfs/Dockerfile

```diff
@@ -5,9 +5,9 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 # Install CDH5 in a single node: Pseudo Distributed
-# Docs: http://www.cloudera.com/content/www/en-us/documentation/enterprise/latest/topics/cdh_qs_yarn_pseudo.html
+# Docs: https://www.cloudera.com/documentation/enterprise/5-6-x/topics/cdh_qs_yarn_pseudo.html 
```

Comment:
Should this be a specific release version?

Reply:
Ideally not, no. However, the old link was dead, and https://www.cloudera.com/documentation/enterprise/latest/topics/cdh_qs_yarn_pseudo.html appears not to be a thing.

Reply:
That's annoying, but OK.